phpBB/includes/functions_display.php

```diff
@@ -547,11 +547,12 @@ function display_forums($root_data = '', $display_moderators = true, $return_mod
 				$last_post_subject = $last_post_subject_truncated = '';
 			}
 			$last_post_time = $user->format_date($row['forum_last_post_time']);
+			$last_post_time_rfc3339 = gmdate("Y-m-d H:i:sP", $row['forum_last_post_time']);
```

Comment:
Why?

Reply:
What’s with the rfc3339 stuff? Are you fixing all date times I. The theme to work with the time and datetime setup in this pr?

Reply:
No no. Just check the ticket description in the issue tracker, and check in the template files where these RFC3339 dates are used. They don't change anything visually, and maintain the ability for users to configure their own date format for output.

Reply:
@hanakin 
I had the same impression I have to admit but that's just a wrapper for the sake of search engines, at the end of all.

Why not to use DATE_ATOM instead? That's the same as of DATE_RFC3339.

https://www.php.net/manual/en/class.datetimeinterface.php#datetime.constants.rfc3339

Reply:
@3D-I You have a point about the format. Maybe we can use some PHP constant. According to MDN documentation for <time>, there are several valid formats. 

I'll recheck the exact format. It looks like the format I'm using is actually not fully RFC3339 (as it's missing the T between date and time now).

Reply:
**RFC3339_EXTENDED** is for PHP >= 7 if I correctly recall?

Reply:
I replaced the date formats with DATE_RFC3339 (available on required PHP versions, and the actual name of the standard). ATOM is a feed standard, which happens to use RFC3339 as its format for dates.